src/stats/common.ts

```diff
@@ -11,6 +11,11 @@ export interface StatsType {
   overall: OverallType[];
 }
 
+export interface StadiumStatsType {
+  targetBreaks: TargetBreakType[] | null;
+  homeRunDistance: number | null;
+}
+
```

Comment:
```suggestion
export type StadiumStatsType = HomeRunContestResultType | TargetTestResultType; 

export interface TargetTestResultType {
  type: 'target-test',
  targetBreaks: TargetBreakType[];
}

export interface HomeRunContestResultType {
  type: 'home-run-contest',
  distance: number;
  units: 'feet' | 'meters';
}

```
The `StadiumStatsType` should probably be discriminated union.